galaxy_ng/app/settings.py

```diff
@@ -134,6 +134,8 @@
 BASE_DIR = os.path.dirname(os.path.dirname(__file__))
 LOCALE_PATHS = (os.path.join(BASE_DIR, 'locale'),)
 
+GALAXY_ENABLE_VIEW_ONLY_ACCESS = os.getenv('GALAXY_ENABLE_VIEW_ONLY_ACCESS', default=False)
```

Comment:
If you default to False here:

```suggestion
GALAXY_ENABLE_VIEW_ONLY_ACCESS = False
```

Then users can overwrite it on `/etc/pulp/settings.py`
It will allow the pulp installer to have 
```yaml
pulp_settings:
  galaxy_enable_view_only_access: true
```

and also

```
export PULP_GALAXY_ENABLE_VIEW_ONLY_ACCESS=true
```